json-rpc/types/src/views.rs

```diff
@@ -554,11 +552,9 @@ impl From<Transaction> for TransactionDataView {
                     script_hash,
                     script_bytes,
                     script: t.into_raw_transaction().into_payload().into(),
-                })
+                }
             }
-        };
-
-        x.unwrap_or(TransactionDataView::UnknownTransaction {})
```

Comment:
why were these removed ?  It is good for backward compatibility

Reply:
because we don't need it, there is no "unknown" transaction, you can see all transaction type is mapped.
it is possible in the future, we will have unknown one, but that will be breaking change to existing code.
in above code, you can see I remove "Ok", so there is no "unwrap_or" anymore.

Technically, I'd like to return error if unwrap failed, instead of return unknown transaction type, which hides real error.
But here we really don't have unknown transaction anymore.